app/src/main/java/zapsolutions/zap/util/TorUtil.java

```diff
@@ -0,0 +1,89 @@
+package zapsolutions.zap.util;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.content.pm.ResolveInfo;
+import android.net.Uri;
+import android.text.TextUtils;
+import android.util.Log;
+
+import java.util.List;
+
+import zapsolutions.zap.R;
+import zapsolutions.zap.connection.establishConnectionToLnd.LndConnection;
+
+public class TorUtil {
+
+    private final static String ORBOT_PACKAGE_NAME = "org.torproject.android";
+    private final static String ORBOT_MARKET_URI = "market://details?id=" + ORBOT_PACKAGE_NAME;
+    private final static String ORBOT_FDROID_URI = "https://f-droid.org/repository/browse/?fdid="
```

Comment:
Not sure, but it looks like this link is not valid anymore.
Maybe we should replace it with this:  `https://guardianproject.info/fdroid/` ?